src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java

```diff
@@ -720,6 +722,23 @@ public static synchronized void registerColumnEncryptionKeyStoreProviders(
                         + globalCustomColumnEncryptionKeyStoreProviders.size());
     }
 
+    /**
+     * Unregisters the custom key store providers from the globalCustomColumnEncryptionKeyStoreProviders.
+     */
+    public static synchronized void unregisterColumnEncryptionKeyStoreProviders() {
+        loggerExternal.entering(SQLServerConnection.class.getName(), "unregisterColumnEncryptionKeyStoreProviders",
+                "Removing Column Encryption Key Store Provider");
+
+        if (null != globalCustomColumnEncryptionKeyStoreProviders) {
+            globalCustomColumnEncryptionKeyStoreProviders.clear();
+        }
+
+        loggerExternal.exiting(SQLServerConnection.class.getName(), "unregisterColumnEncryptionKeyStoreProviders",
```

Comment:
Please do not do this class.getName() in logger, it traverses the stack and is extremely costly performance-wise. I also recommend checking the logging level before calling these entering/exiting methods, although I'm not sure if the intent here is to always log. As for the class.forName(), just use a static string, so SQLServerConnection.class.getName() would be "SQLServerConnection".